### ISSUE TITLE
Tagging ngrams with XSLT

### ISSUE THREAD

**Issue by ebeshero** [amayadwillis/DisneySongs#8]
@amayadwillis I noticed this on your new task list, and was thinking about it. Are these ngrams that you pulled from analysis with Antconc?

If we can make a plain text list of these, we can use `<xsl:analyze-string>` to go looking for them and tag them where they turn up in your collection. @wdjacca and I used this on the [Montage project](https://dxh405.github.io/hughes_project/) last year in DIGIT 110 to look for and tag recurring phrases to highlight in the HTML when they weren't tagged in the XML.

**Comment by ebeshero** [comment]
Anyway I'm happy to help with this, and we even have some starting code to work with from the project last year!

**Comment by amayadwillis** [comment]
Yes they are from antconc! I can work on making a plaintext list of them, I'll try to get it done asap!
